docs/source/release/v3.9.0/imaging.rst

```diff
@@ -0,0 +1,21 @@
+=====================
+Imaging Changes
+=====================
+
+.. contents:: Table of Contents
+   :local:
+
+Tomographic reconstruction graphical user interface
+###################################################
+
+- Running local reconstructions is now possible
+
+Bug Fixes
+---------
+- The external interpreter and scripts paths are no longer ignored and are now correctly appended when running a local reconstruction
```

Comment:
I think that's appeared because I merged the file with #17940, it's not part of this PR